examples/encoding/07_encoding_dash_hls_thumbnail.js

```diff
@@ -0,0 +1,784 @@
+const Promise = require('bluebird');
+
+const BITMOVIN_API_KEY = '<INSERT_YOUR_API_CODE>';
+const bitmovin         = new Bitmovin({'apiKey': BITMOVIN_API_KEY, debug: false});
+
+const ENCODING_NAME = 'encoding_' + (new Date().toISOString());
+
+// Input Settings
+// See https://bitmovin.com/encoding-documentation/bitmovin-api/#/reference/encoding/inputs/create-s3-input for more information
+const S3_ACCESS_KEY   = '<YOUR_S3_ACCESS_KEY>';
+const S3_SECRET_KEY   = '<YOUR_S3_SECRET_KEY>';
+const S3_BUCKET_NAME  = '<YOUR_S3_BUCKET>';
+const S3_CLOUD_REGION = '<YOUR_REGION>';
+const INPUT_FILE_PATH = '/encoding/Sintel-original-short.mkv';
+
+// Output Settings
+// See https://bitmovin.com/encoding-documentation/bitmovin-api/#/reference/encoding/outputs/create-ftp-output for more information
+const FTP_HOST                        = '<YOUR_FTP_HOST>';
+const FTP_USER                        = '<YOUR_FTP_USER>';
+const FTP_PASSWORD                    = '<YOUR_FTP_PASSWORD>';
+const FTP_TRANSFER_VERSION            = '1.1.0';
+const FTP_MAX_CONCURRENT_CONNECTIONS  = 20;
+
+const OUTPUT_PATH     = '/output/bitmovin-javascript-examples/' + ENCODING_NAME + '/';
+const THUMBNAIL_OUTPUT_PATH = OUTPUT_PATH + 'thumbnails/';
+
+console.log('OUTPUT_PATH', OUTPUT_PATH);
+
+const s3Input = {
+  name       :  'Sintel 2010 1080p MKV',
+  description:  'This is a demo s3 input',
+  accessKey:    S3_ACCESS_KEY,
+  secretKey:    S3_SECRET_KEY,
+  bucketName:   S3_BUCKET_NAME,
+  cloudRegion:  S3_CLOUD_REGION
+};
+
+const ftpOutput = {
+  name       :              'FTP Output',
+  description:              'This is a demo FTP output',
+  host:                     FTP_HOST,
+  username:                 FTP_USER,
+  password:                 FTP_PASSWORD,
+  transferVersion:          FTP_TRANSFER_VERSION,
+  maxConcurrentConnections: FTP_MAX_CONCURRENT_CONNECTIONS
+};
+
+// AAC Audio Configuration
+const aacAudioCodecConfiguration       = {
+  name   : 'Audio',
+  bitrate: 128000,
+  rate   : 44100
+};
+
+// Video Codec Configurations
+const h264VideoCodecConfiguration720p  = {
+  name   : 'H264 720p',
+  bitrate: 2400000,
+  rate   : 24.0,
+  width  : 1280,
+  height : 720,
+  profile: 'HIGH'
+};
+
+const h264VideoCodecConfiguration480p = {
+  name   : 'H264 480p',
+  bitrate: 1200000,
+  rate   : 24.0,
+  width  : 854,
+  height : 480,
+  profile: 'HIGH'
+};
+
+const h264VideoCodecConfiguration360p = {
+  name   : 'H264 360p',
+  bitrate: 800000,
+  rate   : 24.0,
+  width  : 640,
+  height : 360,
+  profile: 'HIGH'
+};
+
+const h264VideoCodecConfiguration240p = {
+  name   : 'H264 240p',
+  bitrate: 400000,
+  rate   : 24.0,
+  width  : 426,
+  height : 240,
+  profile: 'HIGH'
+};
+
+const encodingResource = {
+  name: ENCODING_NAME,
+  description: 'bitmovin-javascript api client test encoding',
+  encoderVersion: 'STABLE'
+};
+
+const thumbnailResource = {
+  name: 'My Thumbnail 1',
+  description: 'Demo thumbnail',
+  height: 320,
+  unit: 'SECONDS',
+  positions: [
```

Comment:
Make this configurable on the top of the example. If this is an empty array, omit the thumbnail creation.